### PR TITLE
fix unit tests 

### PR DESCRIPTION
**Pull request by jgebal**

### CODE REVIEW

**Comment by javornikolov** [comment]
Thank you @jgebal! Is that working fine with JRuby (I recall some error message differences in jruby vs mri ruby)?

**Comment by jgebal** [comment]
did not check...

**Comment by jgebal** [comment]
Hmmm.
I just tried to run it on JRuby and got a whole lot of exceptions. Not really related to the change.
```
Java::JavaSql::SQLException: Could not commit with auto-commit set on
oracle.jdbc.driver.PhysicalConnection.commit(oracle/jdbc/driver/PhysicalConnection.java:4439)
oracle.jdbc.driver.PhysicalConnection.commit(oracle/jdbc/driver/PhysicalConnection.java:4486)
java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:606)
RUBY.commit(/home/user/RubymineProjects/ruby-plsql/spec/../lib/plsql/jdbc_connection.rb:65)
RUBY.(root)(/home/user/RubymineProjects/ruby-plsql/spec/plsql/procedure_spec.rb:1848)
ruby.__dash_e__.(root)(-e:1)
ruby.__dash_e__.(root)(ruby/-e:1)
```

File: spec/plsql/procedure_spec.rb [hunk 1]
```diff
@@ -2253,6 +2253,15 @@ def new_candidate(status)
...
 describe '#get_argument_metadata' do
+  before(:all) do
+    plsql.connect! CONNECTION_PARAMS
+  end
+
+  after(:all) do
+    plsql.logoff
+  end
+
+
```
Comment (javornikolov) [inline spec/plsql/procedure_spec.rb]: [Formatting] do we need 2 blank lines here?

Comment (jgebal) [inline spec/plsql/procedure_spec.rb]: Will be fixed

**Comment by jgebal** [comment]
@javornikolov You think it is OK to merge?

**Comment by javornikolov** [comment]
@jgebal, yes, I'm going to merge it. Thank you!

**Pull request merged**